code/game/objects/items/food/misc.dm

```diff
@@ -949,3 +949,11 @@
 	icon_state = "ready_donk_warm_mex"
 	tastes = list("enchiladas" = 2, "laziness" = 1)
 	foodtypes = GRAIN | DAIRY | MEAT | VEGETABLES | JUNKFOOD
+
+/obj/item/food/pierogis
+	name = "pierogis"
+	desc = "Potato and cheese dumplings, straight from Space Kiev. Served with caramelized onions and a healthy dallop of sour cream."
```

Comment:
This is like super super minute... but if this is somewhat in support of Ukraine (with the whole flag thing), I recommend changing Kiev to Kyiv.
```suggestion
	desc = "Potato and cheese dumplings, straight from Space Kyiv. Served with caramelized onions and a healthy dallop of sour cream."
```

Reply:
I've seen it both ways; aren't they more or less interchangeable, and both acceptable?

Reply:
Kiev is the Russian way, Kyiv is the Ukrainian way.